Cargo.toml

```diff
@@ -41,6 +41,10 @@ name = "cmd-program"
 name = "colored"
 required-features = ["colored"]
 
+[[example]]
+name = "pretty-colored"
+required-features = ["colored"]
+
```

Comment:
If it wouldn't be trouble, this example will need `.travis.yml` and `appveyor.yml` entries matching the entries for the `colored` example to ensure it's tested.

Reply:
I never used travis or appveyor, I'll look into it